File: lib/src/media_format/codec.dart
class Codec {
  final String name;

  const Codec(this.name);

  static const AAC = Codec('aac');
  static const OGG = Codec('ogg');
  static const OPUS = Codec('opus');
  static const MP3 = Codec('mp3');
  static const PCM = Codec('pcm');
}

File: sounds/android/src/main/java/com/bsutton/sounds/Codec.java
package com.bsutton.sounds;

// this enum MUST be synchronized with lib/sounds.dart and ios/Classes/SoundsPlugin.h
public enum Codec {
    DEFAULT, AAC, OPUS, CODEC_CAF_OPUS // Apple encapsulates its bits in its own special envelope : .caf instead of a
    // regular ogg/opus (.opus). This is completely stupid, this is Apple.
    , MP3, VORBIS, PCM
}

